### ISSUE TITLE
[ANDROID] Shake Sensitivity

### ISSUE THREAD

**Issue by djhr** [jadsonlourenco/react-native-shake-event#21]
As I've experienced, shake on Android it's a bit more rough than on iOS, sometimes you really have to shake it hard to be detected, as so would be nice to have the ability to set shake sensitivity on Android, as supported by [ShakeDetector.java#L109](https://github.com/square/seismic/blob/master/library/src/main/java/com/squareup/seismic/ShakeDetector.java#L109)

**Comment by jadsonlourenco** [comment]
I agree, PR are welcome: https://github.com/jadsonlourenco/react-native-shake-event/issues/26
